packages/datadog-plugin-kafkajs/test/index.spec.js

```diff
@@ -4,6 +4,12 @@ const { expect } = require('chai')
 const agent = require('../../dd-trace/test/plugins/agent')
 const { expectSomeSpan, withDefaults } = require('../../dd-trace/test/plugins/helpers')
 const { ERROR_MESSAGE, ERROR_TYPE, ERROR_STACK } = require('../../dd-trace/src/constants')
+const Hash = require('../src/hash')
+const sinon = require('sinon')
+
+const DEFAULT_PATHWAY_HASH = Buffer.from('e858292fd15a41e4', 'hex')
+const DEFAULT_PATHWAY_CTX = Buffer.from('e073ca23a5577149a0a8879de561a0a8879de561', 'hex')
+const DEFAULT_TIMESTAMP = 1
```

Comment:
I do like default values for things like these, however the number 1 could be a little dangerous for testing since it's many orders of magnitude off from what a real timestamp would be (such as 1682373336345).

As an example of why that could be dangerous: the number 1 can be represented in a single byte of a buffer, and any bugs pertaining to a buffer being too small might not be discovered by the tests. A bit of a stretch but things like that could happen.

Instead, maybe choose a random date and use the numeric form of that:

```js
const DEFAULT_TIMESTAMP = Number(new Date('2023-04-20T16:20:00.000Z'))
```

Personally I like to sneak in dates like anniversaries or newscasters getting fired :p